sdk/appconfiguration/perf-tests/app-configuration/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@azure-tests/perf-app-configuration",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
can we split this into `dependencies` and `devDependencies`? this keeps coming up and I wonder what is the source of it.

Reply:
We can say all of them are devDependencies since we are writing only tests and no source code.

This was from the wiki guide. How else do you want to see this?

Reply:
see [text analytics](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/textanalytics/perf-tests/text-analytics/package.json) for reference. I thought I already gave feedback about this in the guide PR but maybe it fell through the cracks.

Reply:
I don't care about how we categorize but wanted to understand your mind.
![image](https://user-images.githubusercontent.com/10452642/123341915-a4b6e580-d503-11eb-9d41-2b0bf6c057e3.png)
What is your reasoning here for the categorization?

Reply:
This package is used for testing yes but it is by itself still a package and should be treated as such IMO. Conceptually, any package has source files that could be importing stuff from external packages, those latter ones should be listed in the `dependencies` section. All other dependencies needed for building, testing those source files, and other tooling should go into `devDependencies`.

Reply:
It still feels like not needed.
But I can satisfy you :)